### ISSUE TITLE
To not use pure black (#000) and pure white (#FFF) in the design.

### ISSUE THREAD

**Issue by colomet** [pressbooks/ideas#151]
### Feature Description
To not use pure black (#000) and pure white (#FFF) in the design.

### Feature Use Case
Pure black text and backgrounds with white can cause discomfort for the eye when users read the text over an extended period of time, leading to eye strain.

### Other Notes
http://uxmovement.com/content/why-you-should-never-use-pure-black-for-text-or-backgrounds/

https://graphicdesign.stackexchange.com/questions/25356/why-not-use-pure-black-000-and-pure-white-fff

**Comment by SteelWagstaff** [comment]
Closing for now as won't fix. There are very few places that I'm aware of where Pressbooks displays these colours on top of each other. Pull requests to address any known specific instances would certainly be considered, however.
